package.json

```diff
@@ -21,30 +21,32 @@
     "version": "json -f package.json -I -e \"this.repository.sha = '$(git log -n1 --pretty=format:%H)'\"",
     "watch": "webpack --progress --colors --watch --watch-poll"
   },
+  "dependencies": {
+    "base64-loader": "^1.0.0",
+    "hull.js": "0.2.10",
+    "raw-loader": "^0.5.1",
+    "scratch-render-fonts": "git+https://github.com/LLK/scratch-render-fonts.git",
```

Comment:
Not related to this PR, but maybe we should start publishing these? Any reason not to @thisandagain?